docs/extend/theme.md

```diff
@@ -1 +1,26 @@
-# Quick Start
\ No newline at end of file
+# Themes
+
+Flarum "themes" are just extensions. Typically, you'll want to use the `Frontend` extender to register custom LESS and JS.
+Of course, you can use other extenders too: for example, you might want to support settings to allow configuring your theme.
+
+You can indicate that your extension is a theme by setting the "extra.flarum-extension.category" key to "theme". For example:
+
+```json
+{
+    // other fields
+    "extra": {
+        "flarum-extension": {
+        "category": "theme",
+    // other fields
+}
+```
+
+All this will do is show your extension in the "theme" section in the admin dashboard extension list.
+
+## LESS Variable Customization
```

Comment:
Do we want to use `LESS` or `Less`?

Less themselves style it as `Less`: https://lesscss.org/